src/commands/poll.js

```diff
@@ -53,14 +68,19 @@ exports.run = (client, message, args, callback) => {
             "extendable",
             "straw"
         ],
+        string: [
+            "delayed"
+        ],
         alias: {
             channel: "c",
             extendable: "e",
-            straw: "s"
+            straw: "s",
+            delayed: "d"
         }
     });
 
     let parsedArgs = options._;
+    let delayTime = parseInt(options.delayed, 10);
```

Comment:
using [`parseInt()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) is [discouraged, even with radix](https://github.com/NullDevCo/JavaScript-Styleguide#coercion--numbers). Use `Number(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)` instead.

Reply:
ew

Reply:
nix ew >:c